### PR TITLE
Translate Shallow Renderer

### PR DESCRIPTION
**Pull request by riandy-dimas**
I have done translating this section, kindly review it 🙏

### CODE REVIEW

File: content/docs/addons-shallow-renderer.md [hunk 1]
```diff
@@ -6,31 +6,31 @@ layout: docs
 category: Reference
 ---
 
-**Importing**
+**Mengimpor**
```
Comment (spondbob) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
**Cara Import**
```

Comment (riandy-dimas) [reply content/docs/addons-shallow-renderer.md]: thank you for the suggestion and it's reference 👍

File: content/docs/addons-shallow-renderer.md [hunk 2]
```diff
@@ -42,27 +42,27 @@ const result = renderer.getRenderOutput();
...
 ```
 
-Shallow testing currently has some limitations, namely not supporting refs.
+Saat ini _shallow testing_ masih memiliki beberapa keterbatasan, salah satunya belum mendukung _refs_.
 
-> Note:
+> Catatan:
 >
-> We also recommend checking out Enzyme's [Shallow Rendering API](http://airbnb.io/enzyme/docs/api/shallow.html). It provides a nicer higher-level API over the same functionality.
+> Kami juga menyarankan Anda untuk membaca Enzyme's [Shallow Rendering API](http://airbnb.io/enzyme/docs/api/shallow.html) yang menyediakan _higher-level_ API yang lebih baik untuk fungsionalitas serupa.
```
Comment (regalius) [inline content/docs/addons-shallow-renderer.md]: ... membaca Enzyme's [Shallow Rendering API](http://airbnb.io/enzyme/docs/api/shallow.html)...

... membaca [API Shallow Rendering](http://airbnb.io/enzyme/docs/api/shallow.html) dari Enzyme ...

File: content/docs/addons-shallow-renderer.md [hunk 3]
```diff
@@ -42,27 +42,27 @@ const result = renderer.getRenderOutput();
...
 
-## Reference {#reference}
+## Referensi {#reference}
 
 ### `shallowRenderer.render()` {#shallowrendererrender}
 
-You can think of the shallowRenderer as a "place" to render the component you're testing, and from which you can extract the component's output.
+Anda dapat menggambarkan `shallowRenderer` sebagai “tempat” _render_ suatu komponen yang sedang Anda uji, juga sebagai tempat di mana Anda bisa mengekstrak keluaran dari komponen tersebut.
```
Comment (regalius) [inline content/docs/addons-shallow-renderer.md]: Anda dapat membayangkan `shallowRenderer` sebagai "tempat" untuk me-*render* suatu komponen yang sedang Anda uji...

What do you think? 😄

Comment (riandy-dimas) [reply content/docs/addons-shallow-renderer.md]: sounds good to me 👍

File: content/docs/addons-shallow-renderer.md [hunk 4]
```diff
@@ -6,31 +6,31 @@ layout: docs
 category: Reference
 ---
 
-**Importing**
+**Cara Import**
 
 ```javascript
 import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
 var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
```
Comment (regalius) [inline content/docs/addons-shallow-renderer.md]: Whoops, looks like i missed this, please translate comments and strings inside codeblock 😄

Comment (riandy-dimas) [reply content/docs/addons-shallow-renderer.md]: alright 😄

File: content/docs/addons-shallow-renderer.md [hunk 5]
```diff
@@ -6,63 +6,63 @@ layout: docs
...
 import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
-var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
+var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 dengan npm
 ```
 
-## Overview {#overview}
+## Ikhtisar {#overview}
 
-When writing unit tests for React, shallow rendering can be helpful. Shallow rendering lets you render a component "one level deep" and assert facts about what its render method returns, without worrying about the behavior of child components, which are not instantiated or rendered. This does not require a DOM.
+Ketika menulis _unit tests_ untuk React, _shallow rendering_ akan sangat membantu Anda. _Shallow rendering_ memungkinkan Anda untuk melakukan _render_ suatu komponen “sedalam satu level” dan melakukan pengujian dari hasil kembalian _render method_ komponen tersebut. Hasil _render method_ tidak akan memperdulikan sifat (_behavior_) dari komponen-komponen yang ada di dalamnya, karena komponen-komponen tersebut tidak ter-_render_. Proses ini tidak memerlukan DOM.
```
[the remark this answers is not in the document]
Comment (riandy-dimas) [reply content/docs/addons-shallow-renderer.md]: hi @arfianadam , I have changed this part

**Pull request merged**